bandersnatch/fp/sqrt.go

```diff
@@ -0,0 +1,293 @@
+package fp
```

Comment:
Note this is a new file. We should try to avoid as much as possible "pollute" generated code to avoid headaches in the future.